### DISCUSSION TITLE
How do I get the integration working?

### DISCUSSION THREAD

**Discussion by aaron-saks** [kdeyev/eyeonwater#83] [Q&A]
Hi there.  I am new to HA.  I installed HA today, and added HACS.  I added the custom repository.  You can see below that it shows up.  But I don't see how I can actually add the integration and enter my credentials.  Any help would be great. Thanks!
![image](https://github.com/kdeyev/eyeonwater/assets/56736436/c2541776-47e2-4287-84c7-1896020f09aa)

**Comment by aaron-saks** [comment]
Never mind.  Finally figured it out.

**Reply by cgmarcam** [reply]
hi, can you post the solution? I have the same issue. 

Thanks

**Comment by kdeyev** [comment]
Sorry for the delay. 
Please reach out with any additional questions
